assets/styles/style.css

```diff
@@ -0,0 +1,301 @@
+  @font-face {
+      font-family: 'Myriad Pro Regular';
+      src: url('MyriadPro-Regular.eot');
+      src: url('MyriadPro-Regular.eot?#iefix') format('embedded-opentype'), url('MyriadPro-Regular.woff') format('woff'), url('MyriadPro-Regular.ttf') format('truetype');
+      font-weight: normal;
+      font-style: normal;
+  }
+  
+  .awesome-item {
+      padding-top: 108px;
+      padding-bottom: 105px;
+  }
+  
+  .container {
+      max-width: 1100px;
+      margin-left: auto;
+      margin-right: auto;
+  }
+  
+  .title {
+      position: relative;
+      font-family: 'Raleway', sans-serif;
+      font-size: 36px;
+      text-align: center;
+      color: #212121;
+      margin: 0;
+      padding-top: 0px;
+      padding-bottom: 63px;
+  }
+  
+  .title:after {
+      position: absolute;
+      content: '';
+      display: block;
+      border-bottom: 2px solid #ff8b38;
+      width: 64px;
+      right: 518px;
```

Comment:
also need to define `top` or `bottom` value

and remove `right` value
and add it (later we'll learn it)
```
left:50%;
transformX(-50%);